gaps/__main__.py

```diff
@@ -0,0 +1,4 @@
+from .cli import *
+
+if __name__ == '__main__':
+	cli();
```

Comment:
Please remove the semicolon.

Reply:
I've removed the semicolon and use double quotes instead.